### DISCUSSION TITLE
请问vscode如何点击LESS的（styles.className）跳转到定义？

### DISCUSSION THREAD

**Discussion by ShinChven** [umijs/umi#10858] [Q&A] [answered]
一直使用 IDEA 开发 umijs 项目，可以顺利的通过点击 styles.className 跳转到 LESS 样式的定义，切换到 vscode 以后发现无法通过点击跳转，点className跳转不了，点styles被跳转到了`typings.d.ts`。

请问通过否通过设置 vscode 或者装插件实现这种跳转？

谢谢各位大佬🙏

**Comment by fz6m** [comment, marked as answer]
[CSS Modules](https://marketplace.visualstudio.com/items?itemName=clinyong.vscode-css-modules)

[React Style Helper](https://marketplace.visualstudio.com/items?itemName=iceworks-team.iceworks-style-helper)

装一下这俩。

**Reply by ShinChven** [reply]
好了，跪谢！

**Comment by zhengmenghuang** [comment]
vscode React CSS modules
